### PR TITLE
[IMPROVEMENT] added chrome driver container on docker

### PR DESCRIPTION
**Pull request by taheryasir**
- added env configs database.yml
- added chrome driver to docker
- added configs for spec/rails_helper.rb
- fixed command on docker-compose to fix some issues for docker to run

### CODE REVIEW

File: myapp/docker-compose.yml [hunk 1]
```diff
@@ -13,14 +13,26 @@ services:
...
     volumes:
       - .:/myapp
       - bundle-install:/usr/local/bundle:delegated
     ports:
       - "3001:3000"
     links:
       - db
+    depends_on:
+      - chrome # <-- Link to the chrome container
+    environment: # <-- environment configs for database.yml
```
Comment (yo-it-engineer) [inline myapp/docker-compose.yml:26]: [optional]
`<--` is not necessary. `#` already means "comment"

File: myapp/docker-compose.yml [hunk 2]
```diff
@@ -13,14 +13,26 @@ services:
 
   api:
     build: .
-    command: bash -c "bundle exec rails db:create && bundle exec rails db:migrate && rm -f tmp/pids/server.pid && bundle exec rails s -p 3000 -b '0.0.0.0'"
+    command: bash -c "bundle install && yarn install && bundle exec rails webpacker:install && bundle exec rails db:create && bundle exec rails db:migrate && rm -f tmp/pids/server.pid && bundle exec rails s -p 3000 -b '0.0.0.0'"
```
Comment (yo-it-engineer) [inline myapp/docker-compose.yml:16]: [Memo]
I hope there are other ways of doing this but at least it work and can prevent lots of time lost for new comers.

Reason: running yarn install and bundle install every time when restarting the container takes times. For long term usage, we didn't change gemfile/yarn too often.

File: README.md [hunk 3]
```diff
@@ -232,7 +233,40 @@ chrome://extensions/ を開いて右上のDeveloper modeをオンにして、RKG
   - 今後、PRが大きくなりそうだったらPRを2回以上に分けることを検討しましょう
 
 ### ステップ8: テスト(system spec)を書こう
-
+- First make sure that these gems are exist in Gemfile
```
Comment (yo-it-engineer) [inline README.md:236]: Ikinari English!  lol 

image

File: myapp/docker-compose.yml [hunk 4]
```diff
@@ -13,14 +13,26 @@ services:
...
     ports:
       - "3001:3000"
     links:
       - db
+    depends_on:
+      - chrome # <-- Link to the chrome container
+    environment: # <-- environment configs for database.yml
+      DB_USER: root
+      DB_NAME: myapp
+      DB_PASSWORD: password
```
Comment (yo-it-engineer) [inline myapp/docker-compose.yml:29]: [optional]
Maybe add comment stating that "change these if you want" on the field DB_NAME and DB_PASSWORD.

**Review by yo-it-engineer — APPROVED** [review]
LGTM!

I have leave a few optional suggestions. Feel free to ignore them/update them and can proceed to merge.

PS: please do the same to the branch: `develop_en` too. Will instant LGTM it!

File: myapp/docker-compose.yml (hunk 2 above)
Comment (taheryasir) [reply myapp/docker-compose.yml:16]: true maybe adding these to `Dockerfile` would be better

Comment (taheryasir) [reply myapp/docker-compose.yml:16]: I have tried with two ways, in both ways it takes almost same time (4-5 minutes) to build.

Comment (yo-it-engineer) [reply myapp/docker-compose.yml:16]: well then, let's keep it that way!

**Pull request merged**